pkg/kv/kvserver/asim/workload/workload.go

```diff
@@ -149,6 +152,8 @@ func (rwg *RandomGenerator) Tick(maxTime time.Time) LoadBatch {
 		event := next[key]
 		event.Writes++
 		event.WriteSize += size
+		event.RequestCPU += rwg.requestCPUPerAccess
+		event.RaftCPU += rwg.raftCPUPerWrite
```

Comment:
just confirming, as of this PR, `mmaprototype.RangeLoad.RaftCPU` is not populated anywhere yet, right? I only see one write to it from a datadriven unit test, but nothing in the asim simulations.

Reply:
Lmk if I'm misunderstanding the question. We don't have any integration with mmaprototype yet. 

RangeUsageInfo tracks RaftCPUNanosPerSecond  using the load stats https://github.com/cockroachdb/cockroach/blob/09baa0dc75e6ce13e7919df7fdcd0ab444065741/pkg/kv/kvserver/asim/state/load.go#L96. Load stats get updated https://github.com/cockroachdb/cockroach/blob/09baa0dc75e6ce13e7919df7fdcd0ab444065741/pkg/kv/kvserver/asim/state/load.go#L77. And this field RaftCPU  of LoadEvent is only populated by write workload: 
https://github.com/cockroachdb/cockroach/blob/df2d9b2b1992cb2b055a59d0196b3b03c918fe40/pkg/kv/kvserver/asim/workload/workload.go#L156.